src/login/LoginFailure.jsx

```diff
@@ -168,6 +168,7 @@ const LoginFailureMessage = (props) => {
 
   return (
     <Alert id="login-failure-alert" variant="danger">
+      <Icon src={Info} className="alert-icon" />
       <Alert.Heading>{intl.formatMessage(messages['login.failure.header.title'])}</Alert.Heading>
       <ul>{errorList}</ul>
```

Comment:
remove `<ul>`. Use `<p>`. Since there are no unordered list of items we shouldn't use `<ul>`

Reply:
In other cases of `errorList` still we are using `li` that's why we should not remove `ul`.
We can change this like `{ errorCode === INVALID_FORM ? <ul>{ errorList }</ul> :  { errorList } }`.

Reply:
This is not needed, eventually all error messages will be changed (we are no longer going to use lists for error, see the designs)